src/OpenLoco/Ui/Dropdown.cpp

```diff
@@ -263,17 +263,17 @@ namespace OpenLoco::Ui::Dropdown
                     auto x = _windowDropdownOnpaintCellX * _dropdownItemWidth + self->x + 2;
                     auto y = _windowDropdownOnpaintCellY * _dropdownItemHeight + self->y + 1 + _dropdownItemHeight / 2;
 
-                    if (!(self->getColour(WindowColour::primary) & Colour::translucent_flag))
+                    if (!self->getColour(WindowColour::primary).isTranslucent())
                     {
-                        Gfx::drawRect(*context, x, y, _dropdownItemWidth - 1, 1, Colour::getShade(self->getColour(WindowColour::primary), 3));
-                        Gfx::drawRect(*context, x, y + 1, _dropdownItemWidth - 1, 1, Colour::getShade(self->getColour(WindowColour::primary), 7));
+                        Gfx::drawRect(*context, x, y, _dropdownItemWidth - 1, 1, Colours::getShade(self->getColour(WindowColour::primary).c(), 3));
+                        Gfx::drawRect(*context, x, y + 1, _dropdownItemWidth - 1, 1, Colours::getShade(self->getColour(WindowColour::primary).c(), 7));
                     }
                     else
                     {
-                        uint32_t colour = Colour::getTranslucent(Colour::opaque(self->getColour(WindowColour::primary))) | (1 << 25);
```

Comment:
I like how the original code wanted the opaque colour, just to make it transparent immediately after

Reply:
Yeah technically `getTranslucent` is different to `Colour::translucent` as the first is used to get the palette that represents the translucent version of a colour and the other is just a flag to indicate that the colour is translucent.